ui/apps/everest/src/pages/databases/DbClusterView.tsx

```diff
@@ -325,6 +214,7 @@ export const DbClusterView = () => {
               to="/databases/new"
               variant="contained"
               data-testid="add-db-cluster-button"
+              sx={{ display: canCreate ? 'flex' : 'none' }}
```

Comment:
https://github.com/percona/everest/pull/509#discussion_r1677566500 Referring to this comment, I would like to ask, perhaps it would be better to hide the button altogether?